src/customStyleSystem.js

```diff
@@ -103,6 +103,51 @@ const horizontalRuleColor = style({
   key: 'colors',
 });
 
+const placeholderColor = style({
+  prop: 'placeholderColor',
```

Comment:
why cant we use the already existing color prop?

Reply:
color property can't be applied to a placeholder.
For example. 
I've an input field. if we pass `color` property to it, it will change the color of value, not the placeholder. so, I've added a custom style named `placeholderColor` which would be used to set the placeholder color.

check here how placeholderColor is used. https://github.com/Codebrahma/serverless-design-system/pull/37/files/68baa9bd6c7e26dbd1f91df4354f8367e4fe7e7e#diff-d720dcb43177ec501c5984bedbe8fa95R47

it is used inside `::placeholder` scope.